### ISSUE TITLE
Opentelemery Collector doc crand import alias needs to be mentioned on "check your work" or import section.

### ISSUE THREAD

**Issue by bagmeg** [open-telemetry/opentelemetry.io#3138]
**What needs to be changed?** Describe the update that is required.
From "Building a Trace receiver" tutorial, found that NewSpanID() function is using crand.Reader and it is not mentioned neither inside code (import) nor "check your work" section. I think it is confusing and people will have to make guess which package to use. For myself I used crypto/rand and it worked but not sure if document was meant to use this package.
So I think this should be mentioned somewhere on document.
```go
func NewSpanID() pcommon.SpanID {
	var rngSeed int64
	_ = binary.Read(crand.Reader, binary.LittleEndian, &rngSeed)
	randSource := rand.New(rand.NewSource(rngSeed))
       ...
}
```

**What is the name + path of the page that needs changed?** The relative path
and page title where you found a problem.
content/en/docs/collector/trace-receiver.md

**Comment by svrnm** [comment]
thanks for raising this issue @bagmeg, do you want to provide a PR for fixing this yourself?

**Comment by bagmeg** [comment]
@svrnm sure I can make PR for this. Can you assign it to me?

**Comment by svrnm** [comment]
@bagmeg [We have a policy](https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md#found-a-problem) that we do not assign issues, just go ahead and raise your PR and point out that it fixes this particular issue

**Comment by svrnm** [comment]
fixed via #3140
